backend/image_transfer/encoder.py

```diff
@@ -133,7 +133,7 @@ def create_zip_from_docker_images(
         dest = payload_descriptor.manifests_paths[manifest.docker_image_name]
         zip_file.writestr(dest, manifest.content)
 
-    zip_file.writestr("payload_descriptor.json", payload_descriptor.json(indent=4))
+    zip_file.writestr("payload_descriptor.json", payload_descriptor.model_dump_json(indent=4))
```

Comment:
You need to change also `PayloadDescriptor.parse_raw` into `PayloadDescriptor.model_validate_json` in `decoder.py`

Reply:
Nice catch! You mean [here](https://github.com/Substra/substra-backend/blob/fix/pydantic-updates/backend/image_transfer/decoder.py#L165) and like this, right?
```diff
-     return PayloadDescriptor. parse_raw(zip_file.read("payload_descriptor.json").decode())
+    return PayloadDescriptor.model_validate_json(zip_file.read("payload_descriptor.json").decode())
```

Reply:
yep ! (I did it for the PR refactor image transfer that's why ^^")

Reply:
Also made the pydantic upgrade, but still 🙂

Reply:
Done!